R/lasso-deployment.R

```diff
@@ -25,6 +23,28 @@
 #' F leads to removal of rows containing NULLs.
 #' @param debug Provides the user extended output to the console, in order
 #' to monitor the calculations throughout. Use T or F.
+#' @param cores Number of cores you'd like to use. Defaults to 2.
```

Comment:
This says the cores defaults to 2, but it looks like most of the examples are set to 1... might be good to make the example consistent.